### PR TITLE
GDPR Newsletter Added - Updates to website files inline with theme changes

### PR DESCRIPTION
**Pull request by kylekirkby**

### CODE REVIEW

File: build/README.md [hunk 1]
```diff
@@ -91,7 +90,7 @@ related questions etc. Please see the [MAINTAINERS] file for contact
 information.
 
 <!-- Please keep this list sorted in alphabetic order -->
-<!-- | Platform | Composite PLATFORM flag | Publicly available? |
+| Platform | Composite PLATFORM flag | Publicly available? |
```
[the remark this answers is not in the document]
Comment (kylekirkby) [reply build/README.md]: I've removed the HTML table and left the markdown version.

File: build/README.md [hunk 2]
```diff
@@ -91,7 +90,7 @@ related questions etc. Please see the [MAINTAINERS] file for contact
 information.
 
 <!-- Please keep this list sorted in alphabetic order -->
-<!-- | Platform | Composite PLATFORM flag | Publicly available? |
```
[the remark this answers is not in the document]
Comment (kylekirkby) [reply build/README.md]: This has been removed.

**Comment by kylekirkby** [comment]
@bfletcher merging this PR should result in your blog showing as it has built successfully to https://staging.op-tee.org/blog/security-session-highlights-from-hkg18/

**Comment by jbech-linaro** [comment]
LGTM
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Pull request merged**